GeneralsMD/Code/Libraries/Source/WWVegas/WWMath/vector3.h

```diff
@@ -541,6 +545,18 @@ WWINLINE void Vector3::Lerp(const Vector3 & a, const Vector3 & b, float alpha,Ve
    set_result->Z = (a.Z + (b.Z - a.Z)*alpha);
 }
 
+
+#ifdef ALLOW_TEMPORARIES
+WWINLINE Vector3 Vector3::Lerp(const Vector3 & a, const Vector3 & b, float alpha)
```

Comment:
This function is required for ALLOW_TEMPORARIES. Interestingly this function is mentioned in the comment header of this file, but there was no trace of it. So I have re-added that accordingly.